docs/konnectors.md

```diff
@@ -208,6 +208,114 @@ single thread or application. But NaCl is [no longer
+like me. I could not find any help in any search engine. Some examples are available to run a back in an isolated process and work but I could not run a full nodejs (only nodejs -v worked)
+- The konnectors will need a full nodejs installed on the host
+- Is still actively maintained
+
+### Rkt
+
+Rkt is very similar to docker. It can even directly run docker images from the docker registry, which gives us a lot of existing images to use, even if we want to be able to use other languages than node. For example, we could have also a container dedicated to weboob, another container could use phantomjs or casper and without forcing self-hosted users to do complicated installation procedures.
+
+Rkt is :
+- easy to install : debian, rpm package available, archlinux community package : https://github.com/coreos/rkt/releases
+- has network isolation like docker
+- offers CPU, memory limitation, seccomp isolation (but the set of rules to use is out of my understanding)
+- is well [documented](https://coreos.com/rkt/docs/latest/), complete man pages, but not as well known as docker, then there is not a lot of things to find outside the official documentation.
+- can use docker image directly or can convert them to one runnable aci file with one simple cli command (rkt export)
+- is in active developpement but relatively stable regarding core features.
+- container images can be easily signed and the signature is checked by default when running a container.
+
+I managed to run a nodejs container with just the following commands :
+
+    rkt run --insecure-options=image --interactive docker://node:slim --name nodeslim -- -v
+    rkt list   # to get the container uuid
+    rkt export --app=nodeslim <uuid> nodeslim.aci
+    rkt run --insecure-options=image --interactive nodeslim.aci -- -v  # to run node -v in the new container
+    
+Note: the --insecure-options param is to avoid the check of the image signature to ease the demonstration
+
+### Choice
+
+The best choice would be Rkt for it's ease of use (which is good for contribution) and wide range of isolation features + access to the big docker echosystem without beeing a burden for the host administrator.
+Note : the limitation of NsJail I saw might be due to my lack of knowledge regarding system administration.
+
+### Proposed use of rkt regarding connectors
+
+#### Installation
+
+As stated before, rkt is easy to install. It may also be possible to make it available in the cozy-stack docker image but I did not test it (TODO)
+
+#### Image creation
+
+To create an ACI file image, you just need to run a docker image one time :
+    
+    rkt run  --uuid-file-save=$PWD/uuid --insecure-options=image --interactive docker://node:slim --name nodeslim -- -v
+    rkt export --app=nodeslim `cat uuid` nodeslim.aci && rm uuid
+
+
+The node:slim image weights 84M at this time. The node:alpine image also exists and is way lighter (19M) but I had problems with DNS with this, which could be solved with more time.
```

Comment:
Alpine is not a good idea. It uses a libc that is not the glibc, and it has some nasty side effects: it introduces some bugs that are difficult to track.

cc: @aeris for more on this subject